src/jsonpath.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable camelcase, unicorn/prefer-string-replace-all,
+/* eslint-disable camelcase, jsdoc/valid-types, unicorn/prefer-string-replace-all,
```

Comment:
Please keep the rule enabled, and only disable on a case-by-case basis if really needed. If there is an issue with it, you may just need to add `.js` to the import statements?

Reply:
Done